carts/views.py

```diff
@@ -64,3 +65,29 @@ def delete(self, request, cart_id):
 
         except Cart.DoesNotExist:
           return JsonResponse({'message' : 'CART_NOT_EXISTED'}, status=404)
+
+    @signin_decorator
+    def patch(self, request, cart_id):
+        try:
+            data     = json.loads(request.body)
+            user     = request.user
+            quantity = data['quantity']
+
+            if quantity <= 0:
+                return JsonResponse({'message' : 'QUANTITY_ERROR'}, status=400)
+
+            cart = Cart.objects.get(id=cart_id, user=user)
+
+            cart.quantity = quantity
+            cart.save()
+
+            return JsonResponse({'message' : 'SUCCESS'}, status=201)
```

Comment:
status code만 수정해 주시면 좋을 것 같습니다.
```suggestion
            return JsonResponse({'message' : 'SUCCESS'}, status=200)
```

Reply:
넵 수정하겠습니다!